content/docs/tutorials/deep/reproducibility.md

```diff
@@ -27,8 +27,7 @@ defined in the <abbr>project</abbr>: There's nothing to reproduce.
 $ dvc repro model.p.dvc
 ```
 
-> By default, `dvc repro` tries to read the DVC-file with name `Dvcfile`, like
-> the one we define in the previous chapter.
+> By default, `dvc repro` tries to read all stages available.
```

Comment:
It only tries to read from the `dvc.yaml` from the current directory.

Reply:
OMG yes this tutorial is very outdated. You're totally right but out of scope for this PR. I was just removing the Dvcfile term here, I think. Thanks though!

Reply:
Still, the change you made is incorrect. Please fix or rollback.

Reply:
This is fine don't worry about it. The whole tutorial is going away soon. Resolving. Thanks